src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks.mdx

```diff
@@ -118,13 +116,7 @@ You can also edit the user type and group of a specific user by clicking on that
 
 For how to use our API to manage user information, see [Manage users with NerdGraph](/docs/apis/nerdgraph/examples/nerdgraph-manage-users).
 
-    <img
-      title="Edit user type in the UI"
-      alt="New Relic user management UI - edit user type"
-      src={accountsUserMgmtEditUsers}
-    />
-
-    To manage how users upgrade their user type, see the [authentication domain settings](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#user-upgrade).
+To manage how users upgrade their user type, see the [authentication domain settings](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#user-upgrade).
```

Comment:
Should this be "upgrade their user types" to match the case of the preceding "users"?